packages/cadl-ranch-specs/http/encode/duration/main.tsp

```diff
@@ -0,0 +1,163 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+import "@azure-tools/typespec-client-generator-core";
+
+using TypeSpec.Http;
+using Azure.ClientGenerator.Core;
+
+@doc("Test for encode decorator on duration.")
+@supportedBy("dpg")
+@scenarioService("/encode/duration")
+namespace Encode.Duration;
```

Comment:
Do you think this would belong under the `Type` namespace?

Reply:
I think i like it the current way better, that way we can have a central location for `Encode.Duration`, `Encode.Date`, `Encode.Datetime` etc

Reply:
I don't have a strong opinion, my idea behind that is when you have all the types test the way you serialize them it is important to take the encoding in question

Reply:
but yeah you are probably right, if we keep `Type` as a "yes I can generate this Type"